### PR TITLE
Refactor code, add README

### LINKED ISSUE #11: write readme
include:
1. how to create a new plot
2. the flow of nicola-build
3. where to pipe in data

### PR DESCRIPTION
**Pull request by raivivek**
Closes #11, #6

### CODE REVIEW

File: conf.py [hunk 1]
```diff
@@ -30,8 +33,8 @@
 # This is the URL where Nikola's output will be deployed.
 # If not set, defaults to SITE_URL
 # BASE_URL = "https://tools.wmflabs.org/wigi/"
-BLOG_EMAIL = "vivekrai.iitkgp@gmail.com"
-BLOG_DESCRIPTION = "Demo site for WIGI graph demonstration"  # (translatable)
+BLOG_EMAIL = "vivekraiiitkgp@gmail.com"
```
Comment (hargup) [inline conf.py]: This should rather be the mailing list's email address.

Comment (raivivek) [inline conf.py]: Correct.

File: conf.py [hunk 2]
```diff
@@ -2,10 +2,13 @@
 
 from __future__ import unicode_literals
```
Comment (hargup) [inline conf.py]: From a [recent discussion at SymPy](https://github.com/sympy/sympy/pull/9692#issuecomment-126162173) I learned that we shouldn't use `unicode_literals` and hence this line should be removed.

File: conf.py [hunk 3]
```diff
@@ -2,10 +2,13 @@
...
 import time
-from gender_by_country import create_gender_by_country_plot
-from language_by_gender import create_language_by_gender_plot
-from gender_by_dob import create_gender_by_dob_plot
-from gender_by_culture import create_gender_by_culture_plot
+from plots import (gender_by_country, gender_by_dob, gender_by_culture,
+                   language_by_gender)
+
+#from gender_by_country import create_gender_by_country_plot
```
Comment (hargup) [inline conf.py]: Commenting out these lines doesn't help, please remove them.

Comment (raivivek) [inline conf.py]: Shit.

File: LICENSE [hunk 4]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2014 Max Klein, Vivek Rai, Harsh Gupta
```
Comment (hargup) [inline LICENSE]: `2014` -> `2015`
`Max Klein, Vivek Rai, Harsh Gupta` -> `WIGI Development Team`, you should rather mention our names in the README.

**Comment by hargup** [comment]
Thanks for the README it looks good.

**Pull request merged**